test/parallel/test-http2-create-client-secure-session.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const common = require('../common');
+const fixtures = require('../common/fixtures');
```

Comment:
Nit: would you mind moving this after the `common.hasCrypto` check?

Reply:
Sure.